cap/modules/fixtures/schemas/config.json

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
The `deposit_schema` JSON-schema should be in the `schema` module to be used for validation. Should not be mixed with other JSON-schemas that are used with the deposits

Let's remove extra fields (e.g name, version, is_indexed, etc) and move the file